### ISSUE TITLE
[Meteor] Can't use template events with dropdowns

### ISSUE THREAD

**Issue by coniel** [Dogfalo/materialize#1069]
I'm using a dropdown inside of a template, but when I try and use ```Template.MyTemplate.events``` events are not registered.

Example:

```html
<template name="MyTemplate">
    <button class='dropdown-button btn' href='#' data-activates='dropdown1'>Dropdown</button>
    <!-- Dropdown Structure -->
    <ul id='dropdown1' class='dropdown-content'>
        <li id="one"><a href="#!">One</a></li>
        <li><a href="#!">Two</a></li>
        <li><a href="#!">Three</a></li>
        </ul>
</template>
```

```javascript
Template.MyTemplate.events({
    "click #one": function(event, template) {
        console.log('clicked #one');
    }
});
```

When I click on #one, nothing happens.

I'm guessing this is because the dropdown structure is moved out of the template and placed right before ```</body>``` in the DOM.

I've gotten around it by turning each dropdown item into a template, but it's obviously not a very good solution.

**Comment by Dogfalo** [comment]
In the next release, dropdowns will appear a a child of their activator.

**Comment by coniel** [comment]
Great, thanks!

**Comment by coniel** [comment]
Still having the same issue in Meteor. Dropdowns are not appearing as a child of their activator.

**Comment by coniel** [comment]
Never mind, it was my fault. I still had an older version of the scripts because I had done a temporary fix earlier (#734) and forgotten about it.
